android/app/src/main/res/values/strings.xml

```diff
@@ -212,4 +214,11 @@
     <string name="result_type">Result Type</string>
     <string name="error_create_notification">Error creating notification</string>
 
+    <!--Feed Strings-->
+    <string name="facebook_access_token" templateMergeStrategy="preserve" translatable="false">1786614831603417|DWqP48Lm77hBqCQDWVsq4lFixuo</string>
```

Comment:
How is this generated? If personally, then please remove it. API keys and tokens are not meant to be part of strings

Furthermore, when does it expire?